wirecard/qmore.py

```diff
@@ -74,9 +74,9 @@ def init_datastorage(self, orderIdent,
         return result
 
     def init_frontend(self, amount, currency, paymentType, language,
-                      orderDescription, successUrl, cancelUrl, failureUrl,
-                      serviceUrl, confirmUrl, consumerUserAgent,
-                      consumerIpAddress, autoDeposit=None,
+                      orderDescription, customerStatement, successUrl,
```

Comment:
I think it'll always be filled with the ref of the ticket so I thought it's fine to set it to None.
And we also always set it. Is there another way you'd prefer?